v1/brokers/redis/goredis.go

```diff
@@ -174,15 +174,17 @@ func (b *BrokerGR) Publish(ctx context.Context, signature *tasks.Signature) erro
 	// Adjust routing key (this decides which queue the message will be published to)
 	b.Broker.AdjustRoutingKey(signature)
 
+	delay := signature.Delay
+	signature.Delay = 0
```

Comment:
Could you add a comment here about why we have to set the delay to 0 in the signature.